aten/src/ATen/native/cuda/ScanKernels.cu

```diff
@@ -441,13 +446,51 @@ void scan_innermost_dim(const Tensor& self, Tensor& result, scalar_t init, Binar
   AT_CUDA_CHECK(cudaGetLastError());
 }
 
+#ifdef __HIP_PLATFORM_HCC__
```

Comment:
Is there an issue that we can associate with this workaround? (e.g. as a comment.)

Reply:
I don't think there is. @zasdfgbnm did you file one, or did you just put similar workarounds ad-hoc to TensorIterator kernels?

Reply:
I didn't file any issue.